requirements.txt

```diff
@@ -1,5 +1,6 @@
 setuptools
 requests
 pyyaml
-matplotlib
```

Comment:
Yes, you are totally right. I was confused because `basemap` requires `matplotlib` so if you have one you should have the other but, if the user doesn't want to use `plot_sky` then, `basemap` is not required and therefore `matplotlib` is no granted. I have changed this.